packages/datagrid/src/components/date-cell-renderer/default-date-cell-renderer.component.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+export default function DefaultDateCellRenderer({ data }) {
```

Comment:
Boolean, Date, String renderers can be merged into a DefaultRenderer that would be selected if none is via Inject

Reply:
I created a DefaultRenderer injected by default.

Reply:
Ok great, but that only a half of the change I request : You still have 3 components that are a default renderer duplication. Remove them, so when you get the renderer it gets the default one instead of the duplication.